### PR TITLE
Add path parameter example

### PR DESCRIPTION
**Pull request by maxiscoding28**
Add an example to the documentation using the path parameter.

### CODE REVIEW

**Review by fairclothjm — APPROVED** [review]
LGTM

A few questions: Is https://github.com/hashicorp/vault-action?tab=readme-ov-file#path sufficient or do you think the example is necessary? If you think this is necessary, is there any reason we don't place this example under the https://github.com/hashicorp/vault-action?tab=readme-ov-file#path reference section?

I am trying to understand the motivation of this PR a bit better. 🙂

**Comment by maxiscoding28** [comment]
Yeah let's put it there https://github.com/hashicorp/vault-action?tab=readme-ov-file#path
That makes sense to me.

File: README.md [hunk 1]
```diff
@@ -300,6 +301,20 @@ with:
 If any other method is specified and you provide an `authPayload`, the action will
 attempt to `POST` to `auth/${method}/login` with the provided payload and parse out the client token.
 
+### Custom Path Name
+
+Auth methods at custom path names can be configured using the `path` parameter
```
Comment (fairclothjm) [inline README.md:306]:
```suggestion
Auth methods at custom path names can be configured using the [`path`](#path) parameter
```

**Review by fairclothjm — APPROVED** [review]
LGTM! Thanks @maxiscoding28 !

**Pull request merged**